tests/test_graphql_api/test_reservation/test_create.py

```diff
@@ -113,18 +116,18 @@ def test_reservation__create__use_longest_buffer_times_if_reserving_multiple_res
     #
     reservation_unit_1 = ReservationUnitFactory.create_reservable_now(
         sku="foo",
-        buffer_time_before=datetime.timedelta(),
-        buffer_time_after=datetime.timedelta(),
+        buffer_time_before=timedelta(),
```

Comment:
Wasn't `from datetime import ...` supposed to become a banned import?

Reply:
It's not yet banned _(and having shorter lines allowed for cleaner formatting on some places in the file 😄)_

Reply:
_yet_. I guess it's fine.